src/vso-task-lib/lib/vsotask.js

```diff
@@ -223,3 +256,8 @@ var _filter = function (pattern, options) {
     return minimatch.filter(pattern, options);
 }
 exports.filter = _filter;
+
+var _findFiles = function (pattern) {
+    return glob.sync(pattern);
```

Comment:
I don't think we want to do sync here.  let me check what we did in publish artifacts ...